ready_recipes/tifffile/meta.yaml

```diff
@@ -24,3 +24,8 @@ about:
   home: https://github.com/blink1073/tifffile
   license:  BSD License
   summary: 'Read and write image data from and to TIFF files.'
+
+extra:
+  recipe-maintainers:
+    - blink1073
```

Comment:
Ping @blink1073. Hope you don't mind, but I've put you down as somebody who would be able to contribute meaningfully to the tifffile conda recipe. I'm going to go ahead and merge this PR, but will remove your name if it is a problem.

Cheers!